src/utils/types.ts

```diff
@@ -251,7 +252,7 @@ interface EvaluatorParameters {
 type ElementsById = { [templateElementId: string]: PageElement }
 
 interface Filters {
-  selectedReviewer: number
+  selectedLevel: number
   selectedStage: number
```

Comment:
Maybe change name of this field to "currentStage" since it's no longer selectable?

Reply:
Sure